### DISCUSSION TITLE
How to reset Document Identity to start from 1 again...

### DISCUSSION THREAD

**Discussion by ashokgudur** [ravendb/ravendb#13903] [General]
Hi,

I have many documents that I want to reset the Identity value to restart from 1.
When I copy the database and remove all the documents to fresh start I want to the ID value also start from 1.
However I see the doc Id is resuming from the last value.

I use | (pipe) to generate the Id.

Thanks

**Comment by ashokgudur** [comment]
Well,

I found this in the studio in (#) Identities tab. :)

thanks.

**Comment by ayende** [comment]
You can also do that using the `SeedIdentityOperation`
